Dockerfile

```diff
@@ -1,22 +1,36 @@
+# TODO: make this default to govuk-ruby once it's being pushed somewhere public
 ARG base_image=ruby:2.7.2
-FROM ${base_image}
+ARG app_home=/app
 
-RUN apt-get update -qq && apt-get upgrade -y
```

Comment:
Is there reason why you are not updating the image?
same questions about installing build-essentials and Nodejs packages? are they needed for this app?

Reply:
I added them again but the packages are only updated only at build time so we will need to rebuild frequently to get the updates.